apps/docs/src/app/documentation/core-helpers/stackblitz/stackblitz.service.ts

```diff
@@ -131,6 +131,8 @@ import { Component } from '@angular/core';
     private getFileBasis(file: ExampleFile): string {
         if (file.service) {
             return file.fileName + '.service';
+        } else if (file.pipe) {
```

Comment:
This could be introduced in another PR because other components may use it faster